dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/p2p/PathwaysToParisData.kt

```diff
@@ -5,18 +5,19 @@ import org.dataland.datalandbackend.annotations.DataType
 import org.dataland.datalandbackend.model.p2p.categories.ammonia.P2pAmmonia
 import org.dataland.datalandbackend.model.p2p.categories.automotive.P2pAutomotive
 import org.dataland.datalandbackend.model.p2p.categories.cement.P2pCement
+import org.dataland.datalandbackend.model.p2p.categories.commercialRealEstate.P2pCommercialRealEstate
 import org.dataland.datalandbackend.model.p2p.categories.electricityGeneration.P2pElectricityGeneration
 import org.dataland.datalandbackend.model.p2p.categories.freightTransportByRoad.P2pFreightTransportByRoad
 import org.dataland.datalandbackend.model.p2p.categories.general.P2pGeneral
 import org.dataland.datalandbackend.model.p2p.categories.hvcPlastics.P2pHvcPlastics
 import org.dataland.datalandbackend.model.p2p.categories.livestockFarming.P2pLivestockFarming
-import org.dataland.datalandbackend.model.p2p.categories.realEstate.P2pRealEstate
+import org.dataland.datalandbackend.model.p2p.categories.residentialRealEstate.P2pResidentialRealEstate
```

Comment:
Why do we introduce two classes here? Residential and Commercial real estate are exactly the same. Comparing the files only shows differences in names. This code duplication should be removed.